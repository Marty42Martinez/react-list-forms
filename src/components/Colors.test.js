import React from 'react';
import { shallow } from 'enzyme';
import Colors from './Colors';

describe('Colors component', () => {
  it('renders a Colors', () => {
    const colors = [
      { name: 'color1', hex: '#112233' },
      { name: 'color2', hex: '#332211' }
    ];

    const wrapper = shallow(<Colors colors={colors} />);
    expect(wrapper).toMatchSnapshot();
  });
});