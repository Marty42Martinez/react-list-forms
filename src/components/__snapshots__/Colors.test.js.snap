// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Colors component renders a Colors 1`] = `
<ul>
  <li
    key="color1"
  >
    <Color
      hex="#112233"
      name="color1"
    />
  </li>
  <li
    key="color2"
  >
    <Color
      hex="#332211"
      name="color2"
    />
  </li>
</ul>
`;
